### DISCUSSION TITLE
Error creating a new service from "start from code" template

### DISCUSSION THREAD

**Discussion by galatifederico** [mia-platform/community#378] [Q&A]
### Which section?

Marketplace

### Which is the version?

v12.0.0

### Description

We have run into a schema validation error occurring when we attempt to save an updated configuration that contains a new microservice made from a "start from code" category market place item. We originally noticed this problem when we were attempting to save configurations. We didn’t get any error message and the save did not commit.

**What we have tested:**

- Started with an existing project and a brand new project, for two different Companies
- Used multiple marketplace items
                 - Mia marketplace items: Angular Template of type template, and Go Hello World Example of type example
                 - Custom template
 

**Findings:**

- The only marketplace item that did not produce the error was the custom one (one we built). We have seen other plugins from the marketplace not causing the error, but we did not test outside of that at this time.
- Seems to be isolated to projects that are of the category “start from code”.
- Inspecting the developer console in the web browser when attempting this shows that it is a schema validation error with the message 'should match "then" schema'.
- Adding in breakpoints and stepping through the code, it appears to be erroring either on a model built from the commit message and description, or processing immediately after it. It's difficult to say for certain as the code that was analyzed is what was served to the browser and may not be sufficient to determine the cause.

### Actual Outcome

We create a new service starting from a "start from code" template but the console do not make the commit.

### Expected Outcome

We create successfully a new service starting from a "start from code" template and the changes are saved.

**Comment by pmarchini** [comment]
Hey @galatifederico, 

Thank you for reporting the issue.
Our team is currently investigating this and we aim to identify and resolve the cause promptly.  
We will update you as soon as we have more information or a solution.  

Thanks for your patience while we sort this out.
